urwid/main_loop.py

```diff
@@ -1324,7 +1324,10 @@ def run(self):
         self._loop.set_exception_handler(self._exception_handler)
         self._loop.run_forever()
         if self._exc_info:
-            raise self._exc_info[0](self._exc_info[1]).with_traceback(self._exc_info[2])
+            if PYTHON3:
+                raise self._exc_info
+            else:
+                raise self._exc_info[0](self._exc_info[1]).with_traceback(self._exc_info[2])
```

Comment:
How to remove `__super`? It can't be as simple as replacing every
`self.__super` with `super(<CLASS>, self)`, can it?
If that is really all there is to it, I'm on it.